src/actions/createInitialMessage.ts

```diff
@@ -7,7 +7,9 @@ import { logger } from "../utils/logger";
 import { slackWebClient } from "../utils/slackWebClient";
 
 export const createInitialMessage = async (): Promise<string | void> => {
-  logger.info('START createInitialMessage')
+  const verbose: boolean = core.getInput("verbose");
```

Comment:
do you need getBooleanInput: https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs

otherwise any string is truthy?